packages/docs/site/i18n/tl/docusaurus-plugin-content-docs/current/blueprints/tutorial/01-what-are-blueprints-what-you-can-do-with-them.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Ano ang mga Blueprints?
+slug: /blueprints/tutorial/what-are-blueprints-what-you-can-do-with-them
+description: Learn what Blueprints are and how they configure WordPress Playground. Discover the benefits of using JSON for instant site setup.
```

Comment:
You can translate the description, which is used for SEO, and add a description on social media sharing